examples/applications/semantic-search/semantic_search_quora_elasticsearch.py

```diff
@@ -21,13 +22,17 @@
 from sentence_transformers import SentenceTransformer, util
 import os
 from elasticsearch import Elasticsearch, helpers
+from ssl import create_default_context
 import csv
 import time
 import tqdm.autonotebook
 
 
-
-es = Elasticsearch()
+es = Elasticsearch(
+    hosts=["https://localhost:9200"],
+    basic_auth=("elastic", os.environ["ELASTIC_PASSWORD"],  # displayed at ES server startup
```

Comment:
```suggestion
    basic_auth=("elastic", os.environ["ELASTIC_PASSWORD"]),  # displayed at ES server startup
```
Presumably this was missing?

Reply:
right, thanks! when is moving something last minute ever a good idea? :thinking: :smile: